content/stories/2021/joe-beda.md

```diff
@@ -11,6 +11,7 @@ tags:
 - Ownership
 - BDFL
 - Communitiy
```

Comment:
Shoot, I missed that Community is misspelled -- mind fixing that here and in the other files?

Reply:
Whoops yep will redo now